projects/packages/my-jetpack/_inc/components/product-card/action-button.jsx

```diff
@@ -103,7 +103,9 @@ const ActionButton = ( {
 					onClick: onAdd,
 				};
 			case PRODUCT_STATUSES.ACTIVE: {
-				const buttonText = __( 'View', 'jetpack-my-jetpack' );
+				const defaultButtonText = __( 'View', 'jetpack-my-jetpack' );
+				const statsButtonText = __( 'View detailed stats', 'jetpack-my-jetpack' );
+				const buttonText = slug === 'stats' ? statsButtonText : defaultButtonText;
```

Comment:
Ah yes - I think my main concern was adding an extra conditional. If we can avoid the stats-specific code here then I think the wording "View detailed stats" is great

Reply:
@jboland88, For when the primary "Upgrade" button is not available/showing to the user (in the Stats card), I created a prop, `primaryActionOverride` which allows any product (in this case, Stats), to override/customize the display and behavior of any primary action button, different than the default way it is defined in `action-button.jsx`.  For Stats, we want to change the "View" button to instead display "View detailed stats". This is one way, a small refactor that works for now... 
Although I think ideally each product should define and pass it's own action buttons it wants to display, rather than having them all strictly defined one way for all products in `action-button.jsx`. Although this would be quite a larger refactor, so I thought I would mention it here, so we can plan it for a followup PR.  Your thoughts?